astropy/utils/tests/test_data.py

```diff
@@ -25,6 +25,7 @@
 
 import py.path
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
Oooo is there a secret `_pytest` module? Should we be using a "hidden" module?

Reply:
Probably not. This is part of the Work-In-Progress-ness - I was hoping that using the "official" monkeypatching tools might work more reliably than my own hacks, but they don't. And monkeypatching is officially provided but only in the form of a test fixture, which I don't know how to access from something that isn't a test function. So I had to pull in the secret module to get at the same tools the test fixture uses. Since it doesn't help, if I still need this function once I've fixed the problem I'll do my own manual monkeypatching.